README.md

```diff
@@ -17,7 +25,11 @@ Pass in the path of a Bazel profile on your filesystem as the first argument. Us
 bazel run //cli -- /path/to/bazel_profile.json.gz
 ```
 
+## Contributing
+
+We welcome contributions from the community. Read our [guide to contributing](https://github.com/EngFlow/bazel_invocation_analyzer/blob/main/CONTRIBUTING.md) for details.
+
 ## References
 
-- [Generating a Bazel profile](https://bazel.build/rules/performance#performance-profiling)
-- [Interpreting a Bazel profile](https://bazel.build/rules/performance#profile-information)
+- [Generating a Bazel profile](https://docs.engflow.com/docs/re/faq.html)
```

Comment:
Suggest changing to `https://docs.engflow.com/docs/re/faq.html#how-do-i-capture-a-bazel-profile`... that section is currently at the top of the FAQ, but it might not always be?

Reply:
Good catch and absolutely! I seem to have c&p the wrong link.
https://github.com/EngFlow/bazel_invocation_analyzer/pull/72